versions.yaml

```diff
@@ -355,11 +355,6 @@ externals:
       version: "edk2-stable202411"
       package: "OvmfPkg/OvmfPkgX64.dsc"
       package_output_dir: "OvmfX64"
-    sev:
-      description: "AmdSev build needed for SEV measured direct boot."
-      version: "edk2-stable202411"
-      package: "OvmfPkg/AmdSev/AmdSevX64.dsc"
-      package_output_dir: "AmdSev"
```

Comment:
I thought we decided on the previous PR that ovmf-sev was needed for SNP?

Reply:
It is needed. Please keep it and rename it to SNP

Reply:
We need the kernel hashes functionality